src/test/java/nextstep/subway/acceptance/FavoriteAcceptanceTest.java

```diff
@@ -0,0 +1,240 @@
+                .header("Location")
+                .split("/favorites/")[1];
+
+        // when
+        var deleteResponse = 즐겨찾기_삭제(Long.valueOf(favoriteId));
+
+        // then
+        var favoriteResponse = 즐겨찾기_조회();
+        var favorites = favoriteResponse.jsonPath().getList(".", FavoriteResponse.class);
+        assertAll(
+                () -> assertThat(deleteResponse.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value()),
+                () -> assertThat(favorites).isEmpty()
+        );
+    }
+
+    /**
+     * given 사용자의 즐겨찾기를 등록하고
+     * when 다른 사용자의 ID로 즐겨찾기를 삭제하면
+     * then 에러가 발생한다
+     */
+    @DisplayName("보유하지 않은 즐겨찾기 삭제 실패")
+    @Test
+    void removeFavoriteFailsWhenUserNotMatch() {
+        // given
+        var favoriteId = 즐겨찾기_등록(교대역, 양재역)
+                .header("Location")
+                .split("/favorites/")[1];
+
+        // when
+        var deleteResponse = MemberSteps
+                .givenLogin("user@email.com", "password")
+                    .pathParam("id", favoriteId)
+                .when()
+                    .delete("/favorites/{id}")
+                .then()
+                    .extract();;
+
+        // then
+        assertThat(deleteResponse.statusCode()).isEqualTo(HttpStatus.NOT_FOUND.value());
+    }
+
+    /**
+     * when 등록되지 않은 ID로 즐겨찾기를 삭제하면
+     * then 에러가 발생한다
+     */
+    @DisplayName("존재하지 않는 즐겨찾기 삭제 실패")
+    @Test
+    void removeFavoriteFailsForIdNotExist() {
+        // when
+        var favoriteIdNotExist = 123123L;
+        var deleteResponse = 즐겨찾기_삭제(favoriteIdNotExist);
+
+        // then
+        assertThat(deleteResponse.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    /**
+     * when 인증되지 않은 사용자가 즐겨찾기 경로를 등록하면
+     * then 예외가 발생한다
+     */
+    @DisplayName("생성 API 인증 테스트")
+    @Test
+    void createFavoriteWithAuth() {
+        // when
+        var body = new HashMap<>();
+        body.put("source", 교대역);
+        body.put("target", 강남역);
+
+        var response = RestAssured.
+                given()
+                .body(body)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().log().all()
+                .post("/favorites")
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.UNAUTHORIZED.value());
+    }
+
+    /**
+     * when 인증되지 않은 사용자가 즐겨찾기 경로를 삭제하면
+     * then 예외가 발생한다
+     */
+    @DisplayName("삭제 API 인증 테스트")
+    @Test
+    void deleteFavoriteWithAuth() {
+        // when
+        var favoriteId = 123L;
+        var response = RestAssured
+                .given()
+                    .pathParam("id", favoriteId)
+                .when()
+                    .delete("/favorites/{id}")
+                .then()
+                    .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.UNAUTHORIZED.value());
```

Comment:
상태 응답 확인도 리팩토링 대상이 될 수도 있겠네요 :-) 

```suggestion
      상태_응답_확인(response.statusCode(),HttpStatus.UNAUTHORIZED)
```

Reply:
이 부분도 말씀주신대로 공통화 해보았습니다 ㅎㅎ